the-alexa-skill/typescript/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "the-alexa-skill",
+  "version": "0.1.0",
+  "bin": {
+    "the-alexa-skill": "bin/the-alexa-skill.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "aws-cdk": "1.57.0",
+    "@aws-cdk/assert": "1.57.0",
+    "@types/jest": "^25.2.1",
+    "@types/node": "10.17.5",
+    "jest": "^25.5.0",
+    "ts-jest": "^25.3.1",
+    "ts-node": "^8.1.0",
+    "typescript": "~3.7.2"
+  },
+  "dependencies": {
+    "@aws-cdk/alexa-ask": "^1.57.0",
```

Comment:
Some of these are locked, whilst others aren't. It is best to keep them all locked

Reply:
updated